packages/pelagos/src/table/Table.stories.js

```diff
@@ -441,11 +445,12 @@ export const InlineMenu = (args) => (
 							</TableCell>
 						))}
 						<TableCell>
-							<IconMenu icon={faEllipsisV} flipped>
+							<IconMenu flipped>
```

Comment:
While you are here... I've noticed that the inline `<IconMenu>`'s color on-hover is the same as the row's color on-hover, so it blends in and doesn't look like the menu is specifically hovered.

Not sure if the solution is a layering thing or if the row shouldn't be using on-hover coloring when the `<IconMenu>` is being hovered instead.